tiledb/ml/models/tensorflow_keras.py

```diff
@@ -240,3 +321,82 @@ def _serialize_optimizer_weights(self, include_optimizer: bool = True) -> bytes:
             return pickle.dumps(optimizer_weights, protocol=4)
         else:
             return b""
+
+    def _load_custom_subclassed_model(self, model, model_array):
+
+        if "keras_version" in model_array.meta:
```

Comment:
Is this code taken from Tensorflow Keras?

Reply:
Yes it is part of tensorflow keras loading. I assume you mean the check for `keras version`